students/Ben_freeman/lesson02/series.py

```diff
@@ -0,0 +1,23 @@
+def sum_series(n,m=0,o=1):
+	"Starting with m and o, take the nth term of the sequence generated by recursively summing."
+	if n ==1:
+		return m
+	elif n==2:
+		return o
+	else:
+		return sum_series(n-1,m,o)+sum_series(n-2,m,o)
+def fibonacci(n):
+	"Applies sum_series with m=0,o=1"
```

Comment:
for docstrings use triple double quotes:
```
def fib(n):
    """This is a docstring"""
```

Reply:
Thanks for the feed back, ill make sure to use triple quotes in the future.